apps/sensenet/src/hooks/use-drawer-items.tsx

```diff
@@ -186,7 +186,8 @@ export const useDrawerItems = () => {
         return true
       })
       .then(items => setDrawerItems(items.map(item => getItemFromSettings(item))))
-  }, [getItemFromSettings, repo, repo.security, session, settings])
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
This is not something I'm looking for. In local I don't see the duplicate requests in the develop branch right now. So this is probably safe to revert back.
There are console errors when the getActions request returns with 500 Access denied. Try to log in with editormanatee.
We should catch those errors and log it with the logger.

Reply:
I'm okay with the request count :)

Reply:
❔  Done.  But I think it is not a "user-friendly" solution. 
Now if you log in with editormanatee you will get 3 errors of 'Acces denied'... but you don't know why ..

Reply:
I should have been more clear... This should be logged as debug not as an error.